src/Arcus.Templates.WebApi/Controllers/HealthController.cs

```diff
@@ -3,6 +3,9 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Diagnostics.HealthChecks;
 using GuardNet;
+#if (ExcludeCorrelation == false && ExcludeCorrelation == false)
```

Comment:
This looks off, maybe use this instead?
```suggestion
#if (ExcludeOpenApi == false && ExcludeCorrelation == false)
```